docs/src/index.md

```diff
@@ -12,6 +12,94 @@
 The main focus of the package lies on atmospheric neutrino flux and the neutrino
 propagation through earth.
 
+### Usage
+First of all the basic vacuum properties have to be defined via creating a
+`OscillationParameters` struct with fixed number of neutrino flavours of the 
+considered model.
+```julia
+julia> using Neurthino
+
+julia> osc = OscillationParameters(3);
+```
+The values of the mixing angles, mass squared differences and CP phases are 
+initialised to 0 and have to be set individually.
+```
+julia> osc.mixing_angles[1,2] = 0.59;
+
+julia> osc.mixing_angles[1,3] = 0.15;
+
+julia> osc.mixing_angles[2,3] = 0.84;
+
+julia> osc.cp_phases[1,3] = 3.86;
+```
+The mass squared differences are defined as <img src="http://latex.codecogs.com/gif.latex?\Delta_{ij}=m_i^2-m_j^2"> and 
```

Comment:
We should avoid dependencies from externals sources. I think MathJax should work or not?

\\[ x = \Delta_{ij}=m_i^2-m_j^2 \\]

Reply:
Maybe switch to githubusercontent until GitHub implements this feature: https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b

Reply:
Yeah, I also discovered this GitLab/GitHub difference ... I thought for the docs we could live this external deps